qemu/tests/numa_stress.py

```diff
@@ -138,12 +130,7 @@ def run(test, params, env):
         stress_test.load_stress_tool()
         error_context.context("Get the qemu process memory use status",
                               test.log.info)
-        node_after, memory_after = max_mem_map_node(host_numa_node, qemu_pid)
-        if node_after == most_used_node and memory_after >= memory_used:
-            test.fail("Memory still stick in node %s" % numa_node_malloc)
```

Comment:
Why should this test point be deleted?
The purpose of this patch is to correct the pressure size.
On the other hand, the purpose of this test point is to 
monitor the memory qemu process used should migrate 
from the old node to a new one.

I tested this test point locally and it works as expected.